templates/CRM/Usermenu/Page/UserMenu.tpl

```diff
@@ -0,0 +1,27 @@
+<div class="civicrm-usermenu">
+  <div class="civicrm-usermenu__data">
+    <div class="civicrm-usermenu-card">
```

Comment:
As you are naming things like
`civicrm-usermenu__data`, `civicrm-usermenu__dropdown` etc. Should not this be `civicrm-usermenu__card`?